resources/js/Interfaces/Member.ts

```diff
@@ -0,0 +1,17 @@
+export type Member = {
```

Comment:
This is in a folder called `Interfaces` but it is a type, not an interface. I think it makes sense for it to be a type, but the name of the folder is a bit confusing. I wonder if we should rename the folder to `models` - what do you think?